datanode/gateway/graphql/resolvers.go

```diff
@@ -1338,6 +1338,7 @@ func (r *myPartyResolver) OrdersConnection(ctx context.Context, party *types.Par
 		Pagination: pagination,
 		DateRange:  dateRange,
 		Filter:     filter,
+		MarketId:   marketId,
```

Comment:
(non-blocking): how come marketID is not part of the Filter?

Reply:
The filter is used in both ordersConnection under markets, and parties. It wouldn't make sense to include it there.